Gemfile

```diff
@@ -75,7 +75,10 @@ gem 'webauthn', '~> 2.1'
 gem 'webpacker', '~> 5.1'
 gem 'xmldsig', '~> 0.6'
 gem 'xmlenc', '~> 0.7', '>= 0.7.1'
-gem 'zxcvbn-js'
+
+# This version of the zxcvbn gem matches the data and behavior of the zxcvbn NPM package.
+# It should not be updated without verifying that the behavior still closely matches JS.
+gem 'zxcvbn', '0.1.7'
```

Comment:
it's too bad JSON doesn't support comments, this would be great to keep in our `package.json` too

Reply:
good call. I changed the `package.json` to be locked to a specific version and added the JS version (4.4.2) to the Gemfile comment.

Reply:
not seeing the 4.4.2 in the changes here but 👍 big fan

Reply:
whoops, forgot to push, it's there now.

Reply:
Hey, hello there.
I have just made this PR updating this to version 0.1.9 for a good reason and the newer version still produces same results as dropbox/zxcvbn 4.4.2, just faster now.

Reply:
I don't know if this is the best way to call both of you to see this but sorry if it is not. @mitchellhenke @zachmargolis.

Reply:
Thank you so much, taking a look now!